AEPIdentity/Tests/IdentityStateTests.swift

```diff
@@ -987,6 +1027,56 @@ class IdentityStateTests: XCTestCase {
         XCTAssertTrue(state.identityProperties.isAidSynced == true)
         XCTAssertEqual(1, mockDataStore.dict.count) // identity properties should not be saved to persistence
     }
+
+    func testResetIdentities() {
+        // setup
+        let sharedStateExpectation = XCTestExpectation(description: "Shared state should be updated once")
+
+        let configSharedState = [IdentityConstants.Configuration.EXPERIENCE_CLOUD_ORGID: "test-org",
+                                 IdentityConstants.Configuration.EXPERIENCE_CLOUD_SERVER: "test-server",
+                                 IdentityConstants.Configuration.GLOBAL_CONFIG_PRIVACY: PrivacyStatus.optedIn.rawValue] as [String: Any]
+        state.lastValidConfig = configSharedState
+        let startingEcid = ECID()
+        state.identityProperties.ecid = startingEcid
+        state.identityProperties.advertisingIdentifier = "test-ad-id"
```

Comment:
let's mock the values and assert the reset for all properties being updated: locationHint, blob, customerIds etc.